pinc/project_quick_check.inc

```diff
@@ -63,6 +63,7 @@ function _test_project_for_large_page_images($projectid)
         mysqli_free_result($result_res);
 
         $details .= "</table>";
+        global $page_image_size_limit;
```

Comment:
This isn't used anywhere in this file, leftover from a prior iteration?

Reply:
Used on line 71 and 77 for summary message.

Reply:
Not sure whether this needs to be differentiated or not, but I thought that I'd point out that images have two sizes -- the file size and the image dimensions. There are times when it's appropriate to show one or the other, and there are times when it's appropriate to show both, but "size" by itself may need context.

Reply:
Ah, that's right you moved the declaration. Please move this to the top of the functions - we keep all global variable declarations at the start of a function.

Reply:
Thanks for pointing that out.